### PR TITLE
fix(chat): fix migration issues (Issue #668)

### PR DESCRIPTION
**Pull request by Alexander-Kezik**
- Issue: https://github.com/epam/ai-dial-chat/issues/682

### CODE REVIEW

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-717.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-717.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-717.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-717.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-717.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-717.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat/src/components/Chat/Migration/MigrationFailedModal.tsx [hunk 1]
```diff
@@ -203,14 +225,15 @@ export const MigrationFailedWindow = ({
...
+  const onBackupPrompts = useCallback(() => {
+    dispatch(ImportExportActions.exportLocalStoragePrompts());
+    BrowserStorage.setEntityBackedUp(MigrationStorageKeys.PromptsBackedUp);
+  }, [dispatch]);
 
-  const onRetryWithBackup = useCallback(() => {
-    dispatch(ImportExportActions.exportLocalStorageEntities());
-    retryMigration();
-  }, [dispatch, retryMigration]);
+  const onBackupChats = useCallback(() => {
```
Comment (mikitabut) [inline apps/chat/src/components/Chat/Migration/MigrationFailedModal.tsx]: nit: as I know in react it's usually handleSomething is method inside same component and onSomething is method which passed to current component
What do you think about rename them?

**Review by mikitabut — DISMISSED** [review]

**Review by mikitabut — DISMISSED** [review]

**Review by mikitabut — APPROVED** [review]

**Pull request merged**